.editorconfig

```diff
@@ -126,3 +126,6 @@ dotnet_diagnostic.CA1062.severity = none
 dotnet_diagnostic.CA1303.severity = none
 # Suppressed[CA1001] as not sure how to fix disposable fields within tests
 dotnet_diagnostic.CA1001.severity = none
+dotnet_diagnostic.CA2227.severity = none # list properties should (not?) be readonly
+# dotnet_diagnostic.CA2234.severity = none # call HttpClient.PostAsync(Uri, HttpContent)
```

Comment:
Do you want to remove this line instead of commenting out ?

Reply:
Yes, I absolutely do!